db/deploy/removed-hotel.sql

```diff
@@ -0,0 +1,252 @@
+-- Deploy rams-core:removed-hotel to pg
+
+BEGIN;
+
+CREATE SCHEMA sqitch;
```

Comment:
I'm not sure if running those ``sqitch`` commands even worked properly this time, BTW.  Even though I literally just ran exactly the same commands, it looks like it tried to re-create the ``sqitch`` schema here.  Not sure why.